packages/flutter_tools/gradle/src/main/kotlin/DependencyVersionChecker.kt

```diff
@@ -280,6 +341,33 @@ object DependencyVersionChecker {
             project.logger.error(warnMessage)
         }
     }
+
+    @VisibleForTesting internal fun checkMinSdkVersion(
+        minSdkVersion: MinSdkVersion,
+        project: Project
```

Comment:
Scope nit: consider passing the rootDir path, and the logger then returning if OUT_OF_SUPPORT_RANGE_PROPERTY should be set. 

Project as you can see is a big object that has a lot of scope and we can minimize that scope in this method.